libs/pilight/core/common.c

```diff
@@ -49,6 +49,11 @@
 	#include <sys/mount.h>
 	#include <sys/ioctl.h>
 #endif
+
+#ifndef _WIN32
```

Comment:
I'm wondering because there are lots of places like
```´c
#ifndef _WIN32
#include <regex.h>
```´
FreeBSD is accounted for later in the file, near ```#define use_regex```